docs/DEVELOPING.md

```diff
@@ -30,32 +30,17 @@ For additional operations, see [Usage](https://www.elastic.co/guide/en/enterpris
 
 ### Compatibility
 
-Jira versions 7 or later are compatible with Elastic connector frameworks.
+Jira Cloud or Jira Server versions 7 or later are compatible with Elastic connector frameworks.
```

Comment:
It might also be helpful to explicitly call out that Data Center editions are not currently supported.  @danajuratoni WDYT?

Reply:
@ppf2 Yes, Thanks. I've called it out explicitly.

Reply:
> @ppf2 Yes, Thanks. I've called it out explicitly.

@ppf2 Did you get a chance to look at this change?